app/helpers/repository_helper.rb

```diff
@@ -185,4 +185,13 @@ def news_image(file, format, width = nil, height = nil, options = {})
       weby_file_view(file, format, width, height, options)
     end
   end
+
+  def full_url_image(text, url)
+    position = 0
+    while !text.index("src=",position).nil? do
+      text.insert(text.index("src=", position)+5, url)
+      position = text.index("src=", position)+5
```

Comment:
Surrounding space missing for operator '+'.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.